install.sh

```diff
@@ -72,8 +72,14 @@ function _bash-it_backup_new() {
 
 # Back up existing profile and append bash-it templates at the end
 function _bash-it_backup_append() {
+	local profile_strings=('if [[ -f ~/.profile ]]; then' 'source ~/.profile' 'fi' 'if [[ -f ~/.bashrc ]]; then' 'source ~/.bashrc' 'fi')
 	_bash-it_backup
 	(sed "s|{{BASH_IT}}|$BASH_IT|" "$BASH_IT/template/bash_profile.template.bash" | tail -n +2) >> "$HOME/$CONFIG_FILE"
+	if [[ ! -f ~/.bash_profile ]]; then
+		printf '%s\n\t%s\n%s\n%s\n\t%s\n%s\n' "${profile_strings[@]}" > ~/.bash_profile
```

Comment:
why do we need to use `printf` like this?
this seems like something that could break easily..

Reply:
Oh yeah, that just kinda came out while I was building. I'll put in a more obvious construct 👍

Reply:
Try now?